products/motorola.md

```diff
@@ -0,0 +1,365 @@
+    eol: 2023-06-01
+    link: https://www.motorola.com/us/smartphones-moto-g-stylus-5g/p
+-   releaseCycle: "moto-g200-5g"
+    releaseLabel: "moto g200 5g"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-200-5g/p
+-   releaseCycle: "moto-g100"
+    releaseLabel: "moto g100"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-100/p
+-   releaseCycle: "moto-g82-5g"
+    releaseLabel: "moto g82 5g"
+    support: true
+    releaseDate: 2022-05-01
+    eol: 2025-05-01
+    link: https://www.motorola.com/we/smartphones-moto-g-82-5g/p
+-   releaseCycle: "moto-g71-5g"
+    releaseLabel: "moto g71 5g"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-71-5g/p
+-   releaseCycle: "moto-g62-5g"
+    releaseLabel: "moto g62 5g"
+    support: true
+    releaseDate: 2022-06-01
+    eol: 2025-06-01
+    link: https://www.motorola.com/we/smartphones-moto-g-62-5g/p
+-   releaseCycle: "moto-g60s"
+    releaseLabel: "moto g60s"
+    releaseDate: 2021-06-01
+    eol: 2023-06-01
+    link: https://www.motorola.com/we/smartphones-moto-g-60-s/p
+-   releaseCycle: "moto-g60"
+    releaseLabel: "moto g60"
+    releaseDate: 2021-05-01
+    eol: 2023-05-01
+    link: https://www.motorola.com/be/fr/smartphones-moto-g-60/p
+-   releaseCycle: "moto-g52j-5g"
+    releaseLabel: "moto g52j 5g"
+    releaseDate: 2022-05-01
+    eol: 2024-05-01
+    link: https://www.motorola.co.jp/smartphone-motorola-moto-g52j/p
+-   releaseCycle: "moto-g52"
+    releaseLabel: "moto g52"
+    releaseDate: 2022-04-01
+    eol: 2025-04-01
+    link: https://www.motorola.com/we/smartphones-moto-g-52/p
+-   releaseCycle: "moto-g51-5g"
+    releaseLabel: "moto g51 5g"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-51-5g/p
+-   releaseCycle: "moto-g50-5g"
+    releaseLabel: "moto g50 5g"
+    releaseDate: 2021-08-01
+    eol: 2023-08-01
+    link: https://nz.motorola.com/smartphones-motorola-g50-5g/p
+-   releaseCycle: "moto-g50"
+    releaseLabel: "moto g50"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-50/p
+-   releaseCycle: "moto-g42"
+    releaseLabel: "moto g42"
+    support: true
+    releaseDate: 2022-06-01
+    eol: 2025-06-01
+    link: https://www.motorola.com/we/smartphones-moto-g-42/p
+-   releaseCycle: "moto-g41"
+    releaseLabel: "moto g41"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-41/p
+-   releaseCycle: "moto-g40-fusion"
+    releaseLabel: "moto g40 FUSION"
+    releaseDate: 2021-05-01
+    eol: 2023-05-01
+    link: https://www.motorola.in/smartphones-moto-g40-fusion/p
+-   releaseCycle: "moto-g32"
+    releaseDate: 2022-08-01
+    eol: 2025-08-01
+    support: true
+    link: https://www.motorola.in/smartphones-moto-g32/p
```

Comment:
giving 404

Reply:
Could be replaced by https://www.motorola.com/we/smartphones-moto-g-32/p ?